client/sections.js

```diff
@@ -22,21 +22,20 @@ const extensionSections = extensions.map( extension => {
 	}
 } );
 
-if ( config.isEnabled( 'devdocs' ) ) {
-	sections.push( {
-		name: 'devdocs',
-		paths: [ '/devdocs' ],
-		module: 'devdocs',
-		secondary: true,
-		enableLoggedOut: true
-	} );
+sections.push( {
```

Comment:
I'm not sure generating these bundles if they are disabled at the config level is a good path. It makes extending Calypso and using it to build things harder.